docs/UserGuide.md

```diff
@@ -126,19 +126,20 @@ Examples:
 
 Finds patients whose names contain any of the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find [n/NAME [MORE_NAMES]] [i/NRIC [MORE_NRICS]] [p/PHONE_NUMBER [MORE_PHONE_NUMBERS]] `
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
+* Only the name, nric, and phone number are searched.
 * Only full words will be matched e.g. `Han` will not match `Hans`
 * Patients matching at least one keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 * The respective appointments of patients matching at least one keyword will also be returned. 
+* The command cannot be empty, i.e. `find`. At least one among searching by name, nric, or phone number needs to be given.
 
```

Comment:
Would this sound better?  
`The search requirements cannot be left empty, at least one search requirement (by name, Nric, or phone number) has to be provided`

Reply:
This is definitely better!